### ISSUE TITLE
parameter server is not working model not synced between different worker

### ISSUE THREAD

**Issue by lukehot** [aws-samples/amazon-sagemaker-script-mode#10]
I have a training script that followed the distributed [training script](https://github.com/aws-samples/amazon-sagemaker-script-mode/blob/master/tf-distribution-options/tf-distributed-training.ipynb)  with parameter server approach, 

I found the models seems not synced, every worker is training on it's own data and never communicated to each other. 
by using ShardedByS3Key approach I can validate that only data on the master worker get used. 

So I didn't see the leverage by using 10 /20 workers. seems like every worker just train on it's own data with it's own PS  and save it's own model and never get synced. 

Do I need to add anything in the training code to specify how gradient is synced ?

**Comment by rabowskyb** [comment]
It should not be necessary to add anything special to the training code itself.  

However, due to the fact that distributed training support for TensorFlow 2.x in SageMaker is focusing around Horovod and the native strategies, this particular example will not be refreshed.  Re native strategies, you might check out the example https://github.com/aws-samples/amazon-sagemaker-script-mode/tree/master/tf-sentiment-script-mode in this repo which trains on multiple GPUs on a single instance.
